naresh/src/main/java/com/lftechnology/phpjava/menu/ProgramMenu.java

```diff
@@ -0,0 +1,71 @@
+package com.lftechnology.phpjava.menu;
+
+import com.lftechnology.phpjava.assignmentthree.EvenFibonacci;
+import com.lftechnology.phpjava.assignmentthree.LargestPalindrome;
+import com.lftechnology.phpjava.assignmenttwo.MutiplierChecker;
+import com.lftechnology.phpjava.assignmenttwo.SalutationSplitter;
+
+import java.util.Scanner;
+
+/**
+ * Created by naresh on 5/16/16.
+ */
+public class ProgramMenu {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1118)